src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -185,4 +187,115 @@ private void queueDecisionIssue(Decision decision) {
             sessionEventService.createSessionEvent(decision.getOnlineHearing(), EventTypes.DECISION_ISSUED.getEventType());
         }
     }
+
+    @ApiOperation(value = "Reply to a decision", notes = "A POST request is used to reply to a decision")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Success", response = CreateDecisionReplyResponse.class),
+            @ApiResponse(code = 401, message = "Unauthorised"),
+            @ApiResponse(code = 403, message = "Forbidden"),
+            @ApiResponse(code = 404, message = "Online hearing not found")
+    })
```

Comment:
In the method I could find code for returning statuses:
- 201
- 400
- 404

And none for:
- 401
- 403

Please update the swagger doc tags.